src/main/java/com/youtube/tracker/controller/TrackerController.java

```diff
@@ -21,4 +29,12 @@ public String entryPoint() {
     public ResponseWrapper track() {
         return ResponseWrapper.success();
     }
+
+    @GetMapping("/search")
+    public ResponseWrapper<YoutubeSearchResponse> getChannelDetails(
+            @RequestParam @NotBlank String part,
+            @RequestParam @NotBlank String key,
+            @RequestParam @NotBlank String id) {
+        return ResponseWrapper.success(youtubeTrackerFeign.getChannels(part, id, key).getBody());
```

Comment:
We don't need to expose this, we will just call POST /track which will call both youtube and telegram.
For this PR we do not need any changes in feign

Reply:
ok
I'll remove